cloudflare/src/framework/mod.rs

```diff
@@ -4,14 +4,15 @@ This module controls how requests are sent to Cloudflare's API, and how response
 pub mod apiclient;
 pub mod async_api;
 pub mod auth;
+#[cfg(not(target_arch = "wasm32"))]  // There is no blocking implementation for wasm.
+pub mod blocking_api;
```

Comment:
Can you please add a `pub use blocking_api::*` so that this doesn't break backwards compatibility?

Reply:
I don't think this breaks compatibility since that file only contains "impl" blocks, i.e., no public symbols.